### PR TITLE
Sam/show view

### PR DESCRIPTION
**Pull request by srsexton94**
Hey y'all I'd love your review - clearly haven't gotten much to the promised styling but I think the show action is all set!

I also added the thumbnail to the index view, a bit styling to get them inline 

I'm having trouble with the owner/user part of it though - I'm trying to figure out if we'd be able to perform a get request on the 'users' resource? but I can't quite - keep getting a 404 error

### CODE REVIEW

**Comment by srsexton94** [comment]
Ooh also the update and delete buttons are there - they are non-functional right now

File: assets/scripts/photo/events.js [hunk 1]
```diff
@@ -20,9 +20,20 @@ const onIndexPhotos = function (event) {
...
+  api.showPhoto(id)
+    .then(ui.onShowPhotoSuccess)
+    .catch(ui.onShowPhotoFailure)
+}
+
 const addHandlers = function () {
   $('#photo-create').on('submit', onCreatePhoto)
   $('#photo-index').on('click', onIndexPhotos)
+  $('#index-wrapper').on('click', 'h3', onShowPhoto)
+  $('#index-wrapper').on('click', 'img', onShowPhoto)
```
Comment (ks603) [inline assets/scripts/photo/events.js]: These should probably have a single class that we are using for this event handler. Would avoid having issues with other h3/img elements later, and also let us combine this into one event handler.

File: assets/scripts/templates/app.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+'use strict'
```
Comment (ks603) [inline assets/scripts/templates/app.js]: Looks like you accidentally made a second app.js

File: assets/scripts/templates/helpers/owner.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+'use strict'
```
Comment (ks603) [inline assets/scripts/templates/helpers/owner.js]: I don't think this is the way we're going to be able to get i.e. a user's name from the `owner` of a photo. There's no get route for them, and mongoose has its own functionality for this sort of thing.

See: https://mongoosejs.com/docs/populate.html

File: assets/scripts/templates/show-display.handlebars [hunk 4]
```diff
@@ -0,0 +1,18 @@
+{{#with photo as |photo|}}
+  <section data-id="{{photo._id}}">
+    <img class="full-image" src="{{photo.url}}" alt="photo unavailable">
+    <h3 data-id="{{photo._id}}">
+      {{photo.name}}
+    </h3>
+    <p> Tags:
+      {{#if photo.tags.[0]}}
+        {{photo.tags}}
```
Comment (ks603) [inline assets/scripts/templates/show-display.handlebars]: Should merge Jake's PR with tags-display first, then change this line after a rebase

**Pull request merged**